public/docs/ts/latest/guide/architecture.jade

```diff
@@ -526,7 +510,7 @@ figure
   service with each new instance of that component.
 
   <!-- We've vastly oversimplified dependency injection for this overview.
-  The full story is in the [Dependency Injection](dependency-injection.html) chapter. -->
+  The full story is on the [Dependency Injection](dependency-injection.html) page. -->
```

Comment:
Done. Will use "in" instead of "on". Changing any inconsistencies in capitalization.